Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/FindSimilarIncidentsV2.py

```diff
@@ -108,16 +109,30 @@ def get_incident_labels_map(labels):
     return labels_map
 
 
+def build_similar_keys_list(similar_incident_keys):
```

Comment:
How are the keys similar?
We should consider changing the name and the name of the parameter to something more clear

Reply:
done